.travis.yml

```diff
@@ -8,68 +8,47 @@ php:
 
 env:
   - DRUPAL_CORE=8.1.x
+    CODE_REVIEW=0
+    SIMPLETEST_DB=mysql://root:@127.0.0.1/og
+    SIMPLETEST_BASE_URL=http://localhost:8888
   - DRUPAL_CORE=8.2.x
+    CODE_REVIEW=0
+    SIMPLETEST_DB=mysql://root:@127.0.0.1/og
+    SIMPLETEST_BASE_URL=http://localhost:8888
+  - REVIEW_STANDARD="Drupal"
+    CODE_REVIEW=1
+  - REVIEW_STANDARD="DrupalPractice"
+    CODE_REVIEW=1
+
+matrix:
+  include:
+    - php: 5.5
+      env: CODE_REVIEW=1
 
 mysql:
   database: og
   username: root
   encoding: utf8
 
-before_script:
+before_install:
   # Remove Xdebug as we don't need it and it causes "PHP Fatal error: Maximum
   # function nesting level of '256' reached."
   # We also don't care if that file exists or not on PHP 7.
   - phpenv config-rm xdebug.ini || true
 
-  # Remember the current directory for later use in the Drupal installation.
-  - TESTDIR=$(pwd)
-
-  # Navigate out of module directory to prevent blown stack by recursive module
-  # lookup.
-  - cd ..
-
-  # Create database.
-  - mysql -e 'create database og'
-
-  # Export database variable for kernel tests.
-  - export SIMPLETEST_DB=mysql://root:@127.0.0.1/og
-
-  # Composer.
-  - sed -i '1i export PATH="$HOME/.composer/vendor/bin:$PATH"' $HOME/.bashrc
-  - source $HOME/.bashrc
-
   # Install Composer dependencies.
   - composer self-update
+  - export PATH="$HOME/.composer/vendor/bin:$PATH"
 
-  # Install code sniffer.
-  - composer global require squizlabs/php_codesniffer
-
-  # Coder.
-  - composer global require drupal/coder:dev-8.x-2.x
-  - ln -s ~/.composer/vendor/drupal/coder/coder_sniffer/Drupal ~/.composer/vendor/squizlabs/php_codesniffer/CodeSniffer/Standards/
-  - phpenv rehash
-
-  # Check if there any bad coding standarts
-  - phpcs --standard=Drupal -p --colors $TESTDIR
-
-
-  # Download Drupal 8 core.
-  - travis_retry git clone --branch $DRUPAL_CORE --depth 1 https://git.drupal.org/project/drupal.git
-  - cd drupal
-  - composer install
-
-  # Reference OG in the Drupal site.
-  - ln -s $TESTDIR modules/og
-
-  # Start a web server on port 8888 in the background.
-  - nohup php -S localhost:8888 > /dev/null 2>&1 &
+install:
+  # Remember the current directory for later use in the Drupal installation.
+  - TESTDIR=$(pwd)
 
-  # Wait until the web server is responding.
-  - until curl -s localhost:8888; do true; done > /dev/null
+  - travis_retry $TRAVIS_BUILD_DIR/travis/prepare_tests.sh
+  - $TRAVIS_BUILD_DIR/travis/prepare_coder.sh
 
-  # Export web server URL for browser tests.
-  - export SIMPLETEST_BASE_URL=http://localhost:8888
+  - phpenv rehash
 
 script:
-  # Run the PHPUnit tests which also include the kernel tests.
-  - ./vendor/phpunit/phpunit/phpunit -c ./core/phpunit.xml.dist ./modules/og
+  - $TRAVIS_BUILD_DIR/travis/run_tests.sh
+  - $TRAVIS_BUILD_DIR/travis/run_coder.sh
```

Comment:
Please keep the test steps in the .travis.yml, don't split them off in separate scripts. If this is in separate scripts then we do not get the output on a line-by-line basis in the Travis logs, which makes debugging impossible. It's hard enough to debug things on Travis :)